README.md

```diff
@@ -181,6 +182,7 @@ Flags:
       --no-verification          Don't verify the results.
       --only-verified            Only output verified results.
       --filter-unverified        Only output first unverified result per chunk per detector if there are more than one results.
+      --configFilename           Path to configuration file
```

Comment:
This flag should just be `--config`. You can get the output from `trufflehog --help` and paste it here.

Reply:
Thanks for the pointer, I updated this section with the latest CLI output when using `--help` 👍